src/Console/Application.php

```diff
@@ -52,13 +53,15 @@ private static function checkExtensions()
             "dom",
             "openssl",
             "phar",
-            "zip",
         );
 
         foreach ($required_exts as $ext) {
             if (!extension_loaded($ext)) {
                 Throw new \Exception("Extension '$ext' required but not loaded, full required list: " . implode(", ", $required_exts));
             }
         }
+
+        Archive\Factory::getUnzipperClassName();
```

Comment:
We could also remove those two lines, and letting pickle fail later on, if the zip stuff is required (AFAIK when installing extensions under Windows, and when using the `release` command)